### ISSUE TITLE
A question about the weight of the loss

### ISSUE THREAD

**Issue by syan1992** [tensorlayer/SRGAN#48]
g_gan_loss = 1e-3 * tl.cost.sigmoid_cross_entropy(logits_fake, tf.ones_like(logits_fake), name='g')
    mse_loss = tl.cost.mean_squared_error(net_g.outputs , t_target_image, is_mean=True)
    vgg_loss = 2e-6 * tl.cost.mean_squared_error(vgg_predict_emb.outputs, vgg_target_emb.outputs, 
                                  is_mean=True)

Could you pls tell me the reason that why you set 1e-3 and 2e-6 as the weight of the loss? 
Thanks.

**Comment by zsdonghao** [comment]
it is for balancing, you can find the answer on the paper.

**Comment by normalworld** [comment]
[g_loss = mse_loss + vgg_loss + g_gan_loss](https://github.com/tensorlayer/srgan/blob/8ec76b6d368facb596b9b70782761ca607b652a7/main.py#L99)
Sorry for bothering.I did not mention the author combine the mse_loss and vgg_loss,it will have a better performance?

**Comment by wagamamaz** [comment]
yes, it will have better performance

**Comment by cianohagan** [comment]
The paper states that the VGG feature maps were rescaled by 1/12.75 which is equivalent to multiplying by approx 0.006. The value in the code is 2e-6 or 0.000002. Is there a reason for this?

**Comment by XiaotianM** [comment]
hello, but the regularization loss is not include in this code ?

**Comment by sdlpkxd** [comment]
Excuse me, I was confused by the weights of the losses ,have you ever tried other weights for the losses and how much influence can the weights bring to the results？ Thank you in advance.

**Comment by yuyuziliu** [comment]
Sorry for bothering. Why the VGG feature maps have to be recaled by 1/12.75? I am confused by this.

**Comment by zsdonghao** [comment]
@yuyuziliu  the VGG is pre-trained using image with scale of 0 ~ 1
